### DISCUSSION TITLE
[Question]

### DISCUSSION THREAD

**Discussion by eunsonny-wanted** [wanteddev/montage-web#8] [Q&A]
### 요약

NextJs pages router를 사용하는 프로젝트 환경에서 wds-icon 패키지만 설치해서 사용할 순 없을까요? 🤔

### 상세

NextJs pages router를 사용하는 프로젝트 환경에서 `wds-icon`을 설치해서 사용하려고 했는데요. (다른 패키지 설치 없이 오직 `wds-icon`만을 설치) 
<img width="1238" alt="스크린샷 2024-05-20 오후 6 19 29" src="https://github.com/wanteddev/wds/assets/151716656/a10cd767-66d8-4d6b-be2f-2166b2725e12">
간헐적으로 첨부한 이미지와 같은 에러가 발생하고 있습니다. 추측하기로는 제가 아이콘을 `sx props`를 사용해서 사이즈 조정을 하다보니 관련한 설정이나 의존성이 누락되어서 발생하는 문제인 것 같습니다. 관련해서 어떤 패키지를 추가로 설치해야 할지 고민되는데 `@wanteddev/wds-nextjs` 패키지를 설치하고 문서대로 세팅해주면 될까요? 혹시 `wds-icon`만 패키지만 단독으로 설치해서 사용할 수 있는 방법은 없을지 👀 문의 드려봅니다!

**Comment by Sh031224** [comment]
`@wanteddev/wds-icon` 에서도 sx prop (내부에서 theme 사용)을 이용하기 때문에 context 오류가 발생하는 것 같습니다.

`@wanteddev/wds`, `@wanteddev/wds-nextjs` 패키지 설치 후 `ThemeProvider` 셋업을 진행해야 합니다. 
icon만 별도로 사용하는 방법은 방법을 검토해보겠습니다!

**Reply by Sh031224** [reply]
`@wanteddev/wds-nextjs`, `@wanteddev/wds-icon` 두 패키지로만 동작하도록 수정되었습니다.
